packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -1,19 +1,128 @@
 import Web3 from "web3";
 import winston from "winston";
+import { getAbi, ExpandedERC20Web3 } from "@uma/contracts-node";
+
+import { InsuredBridgeL1Client, InsuredBridgeL2Client, Deposit } from "@uma/financial-templates-lib";
+
+enum RelayerMode {
+  OnlySlow,
+  OnlySpeedUp,
+  Any,
+}
+
+enum RelayType {
+  Slow,
+  Fast,
+  No,
```

Comment:
Is there a better Type name than "No"?

Reply:
I hope so. what do you recommend? `NotProfitable` or `DontRelay`?

Reply:
Yeah maybe `DontRelay` or `IgnoreRelay` would be more explicit

Reply:
used `IgnoreRelay`